### DISCUSSION TITLE
How do I get the window object?

### DISCUSSION THREAD

**Discussion by JangCool** [nuxt/nuxt#18249] [Questions] [answered]
I'm trying to develop in a SPA way.

I want to use the window object provided by browser, but it doesn't seem to be called in nuxt3.

I want to call a function like window.open(). What should I do?

**Comment by wobsoriano** [comment, marked as answer]
```ts
if (process.client) {
    window.open()
}

// or

onMounted(() => {
    window.open()
})
```
